modules/client/shared/src/main/scala/lucuma/itc/client/decoders.scala

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.client
+
+import cats.Eq
+import cats.syntax.all.*
+import eu.timepit.refined.types.numeric.*
+import io.circe.Decoder
+import io.circe.DecodingFailure
+import io.circe.Json
+import io.circe.JsonObject
+import lucuma.core.data.Zipper
+import lucuma.core.data.ZipperCodec.given
+import lucuma.core.math.SignalToNoise
+import lucuma.core.util.TimeSpan
+import lucuma.itc.AsterismIntegrationTimes
+import lucuma.itc.IntegrationTime
+import lucuma.itc.TargetIntegrationTime
+
+given Decoder[IntegrationTime] = c =>
```

Comment:
Do we have tests for these?

Reply:
I haven't gotten around to tests for the new use cases and classes yet.